ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -1656,6 +1656,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-sdn
+- as: ibmcloud-ovn-multi-ppc64le-f7
+  cron: 20 17 * * *
```

Comment:
The test name states `-f7` (which means the test is with frequency f7, which runs run every 7 days), but the cron entry show it runs every day.  I'm not sure which one is expected here.